libpod/runtime_pod_infra_linux.go

```diff
@@ -34,6 +35,9 @@ func (r *Runtime) makeInfraContainer(ctx context.Context, p *Pod, imgName, imgID
 	// Set Pod hostname
 	g.Config.Hostname = p.config.Hostname
 
+	// Indicate that this container is managed by podman
+	g.AddAnnotation(annotations.ContainerManager, annotations.ContainerManagerLibpod)
```

Comment:
Not necessary - the line on 341 will fire for all containers, even infra containers

Reply:
Okay, I remove it.